Sources/Testing/EntryPoints/EntryPoint.swift

```diff
@@ -439,7 +439,9 @@ extension Event.ConsoleOutputRecorder.Options {
     var result = Self()
 
     result.useANSIEscapeCodes = _fileHandleSupportsANSIEscapeCodes(fileHandle)
-    if result.useANSIEscapeCodes {
+
+    // Respect the FORCE_COLOR environment variable. SEE: https://www.force-color.org
+    if result.useANSIEscapeCodes || Environment.variable(named: "FORCE_COLOR")?.isEmpty == false {
```

Comment:
What is the intended interaction between `FORCE_COLOR` and `NO_COLOR` if both are specified?

Reply:
The NO_COLOR environment variable takes precedence.